node/test/streaming_bisect.js

```diff
@@ -234,7 +234,18 @@ var search = clusterSearch.ClusterIsolateSearch(extend({
                 serviceName: 'test_as_raw',
             }
         });
-        callback(null);
+        var options = cluster.testRawClient.requestOptions({
```

Comment:
You shouldn't need any of these options, in fact you really don't even need to use choosePeer, rather:
```
var peers = cluster.testRawClient.peers.values();
var ready = new CountedReadySignal(peers.length);
peers.forEach(function each(peer) {
  peer.connect().on('identified', ready.signal);
});
ready(callback);
```